witness.schema.json

```diff
@@ -0,0 +1,393 @@
+          ]
+        }
+      }
+    },
+    "target": {
+      "description": "Target entry of certification.",
+      "type": "object",
+      "required": [
+        "uuid",
+        "type",
+        "file_hash"
+      ],
+      "properties": {
+        "uuid": {
+          "$ref": "#/$defs/entry-uuid",
+          "description": "Unique identifier of the target entry."
+        },
+        "type": {
+          "type": "string",
+          "description": "`entry_type` of the target entry."
+        },
+        "file_hash": {
+          "$ref": "#/$defs/hash",
+          "description": "SHA-256 hash of the target file."
+        }
+      }
+    },
+    "certification": {
+      "description": "Certification result.",
+      "type": "object",
+      "required": [
+        "string",
+        "type",
+        "format"
+      ],
+      "properties": {
+        "string": {
+          "type": "string",
+          "description": "The certification result itself."
+        },
+        "type": {
+          "description": "Interpretation of `string`. The following values are supported:\n\n* `verdict`: Verdict about the targeted entry.",
+          "enum": [
+            "verdict"
+          ]
+        },
+        "format": {
+          "description": "Format of `string`. The following values are supported:\n\n* `confirmed | rejected`: Either \"confirmed\" or \"rejected\".",
+          "enum": [
+            "confirmed | rejected"
+          ]
+        }
+      }
+    }
+  },
+  "type": "array",
+  "items": {
+    "oneOf": [
+      {
+        "title": "Loop invariant",
+        "description": "Loop invariants are important building blocks in software verification. Many verification approaches use loop invariants as lemmata to construct proofs of correctness.",
+        "type": "object",
+        "required": [
+          "entry_type",
+          "metadata",
+          "location",
+          "loop_invariant"
+        ],
+        "properties": {
+          "entry_type": {
+            "const": "loop_invariant"
+          },
+          "metadata": {
+            "$ref": "#/$defs/metadata"
+          },
+          "location": {
+            "$ref": "#/$defs/location",
+            "description": "Location in the source code where the loop invariant holds."
+          },
+          "loop_invariant": {
+            "$ref": "#/$defs/invariant",
+            "description": "The loop invariant itself."
+          }
+        },
+        "examples": [
+          {
+            "entry_type": "loop_invariant",
+            "metadata": {
+              "format_version": 0.1,
+              "uuid": "91023a0f-9f45-4385-88c4-1152ade45537",
+              "creation_time": "2021-05-05T13:18:43.000Z",
+              "producer": {
+                "name": "CPAchecker",
+                "version": "2.0.1-svn",
+                "configuration": "(Optional) svcomp21--04-kInduction",
+                "description": "(Optional)",
+                "command_line": "(Optional)"
```

Comment:
Having these `(Optional)` entries in the example is confusing. The generated HTML documentation already clearly states which fields are required, so I would suggest to remove `description` and `command_line` completely and also remove the `(Optional) ` prefix from `configuration`.

Same in the `loop_invariant_certificate` example.